### PR TITLE
Add: scroll function, max items to show, input name props

### PR DESCRIPTION
**Pull request by DeVoresyah**
PR for Issue #2 

## Usage:
```html
<Dropdown
    name="zipcode"
    :maxItem="10">
</Dropdown>
```

### Options
- `name` (optional): An input name | default: `dropdown`
- `maxItem` (optional): Max item to show | default: `10`

### CODE REVIEW

File: src/Dropdown.vue [hunk 1]
```diff
@@ -45,6 +53,12 @@
         required: false,
         default: false,
         note: 'Disable the dropdown'
+      },
+      maxItem: {
+        type: Number,
+        required: false,
+        default: 10,
```
Comment (romainsimon) [inline src/Dropdown.vue:60]: You should change the default to 6 to keep it consistent with previous version and avoid having a scrollbar by default 
```suggestion
        default: 6,
```

File: README.md [hunk 2]
```diff
@@ -47,11 +49,13 @@ Once installed, it can be used in a template as simply as:
 - `options` (required): An array of options with `id` and `name`
 - `placeholder` (optional): A placeholder 
 - `disabled` (optional): true/false
+- `name` (optional): An input name | default: `dropdown`
+- `maxItem` (optional): Max item to show | default: `10`
```
Comment (romainsimon) [inline README.md:53]:
```suggestion
- `maxItem` (optional): Max item to show | default: `6`
```

**Review by romainsimon — CHANGES_REQUESTED** [review]
Looks good, but I think we could let the current default value

File: src/Dropdown.vue (hunk 1 above)
Comment (DeVoresyah) [reply src/Dropdown.vue:60]: no probs

**Comment by romainsimon** [comment]
@DeVoresyah Can you commit my suggested changes ? :)

**Comment by DeVoresyah** [comment]
> @DeVoresyah Can you commit my suggested changes ? :)

yes I'll, need a time. Don't have free time for a while

**Comment by romainsimon** [comment]
I merged your PR and fixed this here https://github.com/romainsimon/vue-simple-search-dropdown/pull/7

**Comment by DeVoresyah** [comment]
> I merged your PR and fixed this here #7

thanks and sorry, I have not had time to change it

**Comment by romainsimon** [comment]
No problem ;)

**Pull request merged**